.jenkins/caffe2/test.sh

```diff
@@ -8,6 +8,11 @@ if [[ "${BUILD_ENVIRONMENT}" == *-android* ]]; then
   exit 0
 fi
 
+if [[ $BUILD_ENVIRONMENT == *-rocm* ]]; then
+  # TODO: add this to docker image
+  export HCC_LAZYINIT=ON
```

Comment:
It's 'ON', not 1?

Reply:
Comes from amd folks https://github.com/pytorch/pytorch/pull/18640#issuecomment-489152269
Locally tried "ON", I believe it's on as long as the environment variable is set.

Reply:
lol https://github.com/RadeonOpenCompute/hcc/blob/e72465c/lib/mcwamp.cpp#L418-L423 it's literally testing for "ON" :-)